### PR TITLE
fix: registrations

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Function.cs [hunk 1]
```diff
@@ -82,9 +85,9 @@ protected override IServiceProvider ConfigureServices(IServiceCollection service
 
             builder
                 .RegisterModule(new DataDogModule(configuration))
-                .RegisterModule<EnvelopeModule>()
-                .RegisterModule(new EditModule(configuration, services, loggerFactory))
+                .RegisterModule(new CommandHandlingModule(configuration))
                 .RegisterModule(new BackOfficeModule(configuration, services, loggerFactory))
+                .RegisterModule(new EventHandlingModule(typeof(DomainAssemblyMarker).Assembly, eventSerializerSettings))
```
Comment (jvandaal) [inline src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Function.cs:90]: I think event handling is registered through CommandHandlingModule -> AggregateSourceModule.

File: src/BuildingRegistry.Consumer.Address/Infrastructure/Program.cs [hunk 2]
```diff
@@ -148,7 +149,8 @@ public static async Task Main(string[] args)
 
                     builder
                         .RegisterModule(new DataDogModule(hostContext.Configuration))
-                        .RegisterModule(new EditModule(hostContext.Configuration, services, loggerFactory))
+                        .RegisterModule(new CommandHandlingModule(hostContext.Configuration))
+                        .RegisterModule(new EventHandlingModule(typeof(DomainAssemblyMarker).Assembly, EventsJsonSerializerSettingsProvider.CreateSerializerSettings()))
```
Comment (jvandaal) [inline src/BuildingRegistry.Consumer.Address/Infrastructure/Program.cs:153]: Same comment as in the Lambda Function.cs.

File: src/BuildingRegistry.Infrastructure/Modules/AggregateSourceModule.cs [hunk 3]
```diff
@@ -36,6 +35,8 @@ protected override void Load(ContainerBuilder builder)
                 snapshotStrategy = IntervalStrategy.SnapshotEvery(snapshotInterval);
             }
 
+            builder.RegisterSnapshotModule(_configuration);
```
Comment (jvandaal) [inline src/BuildingRegistry.Infrastructure/Modules/AggregateSourceModule.cs:38]: I think we are missing EventHandlingModule, see
https://github.com/Informatievlaanderen/streetname-registry/blob/main/src/StreetNameRegistry.Infrastructure/Modules/AggregateSourceModule.cs#L41

File: src/BuildingRegistry.Migrator.Building/Infrastructure/Modules/ApiModule.cs [hunk 4]
```diff
@@ -46,8 +48,9 @@ protected override void Load(ContainerBuilder builder)
 
             builder
                 .RegisterModule(new DataDogModule(_configuration))
-                .RegisterModule<EnvelopeModule>()
-                .RegisterModule(new EditModule(_configuration, _services, _loggerFactory))
+                .RegisterModule(new SequenceModule(_configuration, _services, _loggerFactory))
+                .RegisterModule(new CommandHandlingModule(_configuration))
+                .RegisterModule(new EventHandlingModule(typeof(DomainAssemblyMarker).Assembly, EventsJsonSerializerSettingsProvider.CreateSerializerSettings()))
```
Comment (jvandaal) [inline src/BuildingRegistry.Migrator.Building/Infrastructure/Modules/ApiModule.cs:53]: Same comment as in the Lambda Function.cs.

File: src/BuildingRegistry.Infrastructure/Modules/AggregateSourceModule.cs (hunk 3 above)
Comment (pgallik) [reply src/BuildingRegistry.Infrastructure/Modules/AggregateSourceModule.cs:38]: line 48

**Review by ArneD — APPROVED** [review]

**Pull request merged**